### PR TITLE
Added archive message and disabled Github actions

### PR DESCRIPTION
**Pull request by donpui**
Added not and disabled github actions. Later on we can change repository setting to become archived

---

## Code Review Checklist (to be filled out by reviewer)

- [ ] Description accurately reflects what changes are being made.
- [ ] Description explains why the changes are being made (or references an issue containing one).
- [ ] The PR appropriately sized.
- [ ] New code has enough tests.
- [ ] New code has enough documentation to answer "how do I use it?" and "what does it do?".
- [ ] Existing documentation is up-to-date, if impacted.

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -1,81 +1,83 @@
...
+#         uses: actions/setup-python@v2
+
+#       - name: install mailbox server
+#         run: pip install magic-wormhole-mailbox-server
+
+#       - name: install relay server
+#         run: pip install git+https://github.com/magic-wormhole/magic-wormhole-transit-relay.git
+
+#       - name: run mailbox server
+#         run: twist wormhole-mailbox --usage-db=usage.sqlite &
```
[the remark this answers is not in the document]
Comment (donpui) [reply .github/workflows/main.yml:55]: So this is old repo, which we will archive, should we make here any changes?

[the remark this answers is not in the document]
Comment (meejah) [reply .github/workflows/main.yml:55]: oh, i see it's all just commented-out. just delete instead of commenting, I'd say...

File: README.md [hunk 2]
```diff
@@ -1,4 +1,6 @@
-# Transfer
+## **Note:** This repository is not maintened anymore. Frontend application code was refactored and new version is available at [Winden repository](https://github.com/LeastAuthority/winden). Be aware that packages are outdated and may have security issues.
```
Comment (on-ba) [inline README.md:1]: Suggestion: 

> This repository is not maintained anymore. Be aware that the packages are outdated and may have security issues. The front-end application code was refactored and a new version is available as the [Winden](https://github.com/LeastAuthority/winden) repository.

**Review by on-ba — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (donpui) [reply README.md]: Ok

**Review by on-ba — APPROVED** [review]

**Pull request merged**